src/components/common/forms/ContactForm.tsx

```diff
@@ -11,11 +11,11 @@ const ContactForm = () => {
   const {
     register,
     handleSubmit,
-    formState: { errors },
+    formState: { isValid, isSubmitted },
   } = useForm();
 
   const dispatch = useAppDispatch();
-  const isFormValid = Object.keys(errors).length === 0;
+  // const isFormValid = Object.keys(errors).length === 0;
```

Comment:
Remember to remove this commented line

Reply:
I see you're always using the same logic for checking the validity. You could add a const for that:
`const isFormValid = isSubmitted && isValid;`

Reply:
Lovit, I'll apply it in the opened PR